### DISCUSSION TITLE
After ionic v6 update, apk build breaks

### DISCUSSION THREAD

**Discussion by kowabungaboy** [ionic-team/capacitor#5383] [Q&A]
Hi everyone,

I have asked this already in the [official ionic forum](https://forum.ionicframework.com/t/after-ionic-v6-update-apk-build-breaks/219444) but so far, I haven't received any input so I'm trying my luck here. I am not able to build a functioning apk with the newest ionic v6. The steps to recreate are straight forward, create a new project, add android, build an apk and try to run it on any android device. I have noticed an Error stating `E/InstallStart: Requesting uid 10082 needs to declare permission android.permission.REQUEST_INSTALL_PACKAGES`, but I can't figure out where to place the permissions, nothing seems to work. Also Android Studio tells me the generated permissions should be before the application tag in the manifest, I tried that, it worked once, but I haven't been able to recreate the positive result anymore. Any hint is greatly appreciated.

Thanks in advance and kind regards

**Comment by jcesarmobile** [comment]
The android.permission.REQUEST_INSTALL_PACKAGES is used to install apps from another app, so you shouldn’t include it in your app, unless your app installs other apps.
Updating ionic should not affect capacitor, capacitor doesn’t care about what framework you use.
Have you also updated other things like android SDK, SDK tools, and similar things?
Have you tried running the app from Android Studio?

**Reply by kowabungaboy** [reply]
I have the following android studio: Android Studio Arctic Fox | 2020.3.1 Patch 4
Build tools is 31, same goes for the sdk
the app can be run as debug or as run with android studio, only the apk can't be installed. Also, it doesn't matter  if the apk is signed or not.

The persmissions are also already present from the capacitor-cordova-android-plugins

Thanks and kind regards

**Reply by kowabungaboy** [reply]
Update: The App works again on the testers phone after a factory reset, which is fine. Although it would be interesting to know, why that happened. Anyway Thanks for the support!
